1.0/docs/devguide/styling.md

```diff
@@ -342,112 +421,195 @@ Example usage of `my-toolbar`:
 
 ### Custom property API for {{site.project_title}} elements {#style-api}
 
-{{site.project_title}}'s custom property shim evaluates and applies custom property values once
-at element creation time.  In order to have an element (and its subtree) re-
-evaluate custom property values due to dynamic changes such as application of
-CSS classes, etc., call the [`updateStyles`](../../api/#Polymer.Base:method-updateStyles){:target="_blank"}
-method on the element. To update all elements on the page, you can also call
-`Polymer.updateStyles`.
+You can directly modify a {{site.project_title}} element's custom property by
+setting key-value pairs in
+[`customStyle`](../../api/#Polymer.Base:property-customStyle) on the element.
+This is analogous to setting `style` on an element.
 
-You can  directly modify a {{site.project_title}} element's custom property by setting
-key-value pairs in [`customStyle`](../../api/#Polymer.Base:property-customStyle) on the
-element (analogous to setting `style`) and then calling `updateStyles`. Or you can pass
-a dictionary of property names and values as an argumen to `updateStyles`.
+Note that in shady DOM you then need to call `updateStyles()` in order for the
```

Comment:
This is a limitation of the custom properties shim and is not related to shady DOM.